.github/workflows/create-release-branch.yml

```diff
@@ -66,27 +68,69 @@ jobs:
           else
             RELEASE_TYPE=minor
           fi
-          # Increment the version as needed
+          # Increment the version as needed.
           npm --no-git-tag-version version $RELEASE_TYPE
-          git add package.json
-      # Update the dependencies
-      - name: Update dependencies
-        run: |
-          yarn upgrade
-          node tools/UpdateRetireJsVulns.js
-          git add yarn.lock
-          git add retire-js
-      # Create the new branch
-      - name: Checkout new branch, commit, push
-        id: create-branch
+      # Create the new branch as a direct copy of `dev` and push it to GitHub so the API can
+      # interact with it later.
+      - id: create-branch
         run: |
           NEW_VERSION=$(jq -r ".version" package.json)
+          git stash
+          git checkout dev
```

Comment:
I thought the workflow is already running from dev's latest. Why do we need to checkout dev and do a git pull?

If we didn't do this, then we wouldn't need to stash either. We would just increment the version and then checkout the new branch which forwards our changes automatically. Or am I missing something?